buildenv/jenkins/variables/defaults.yml

```diff
@@ -157,8 +162,9 @@ boot_jdk_default:
       11: '11'
       17: '17'
       21: '21'
-      22: '21'
-      next: '21'
+      22: '22'
```

Comment:
I think jdk22 should continue to boot with jdk21.

Reply:
Updated.